modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-0_{context}"]
+= Release notes for {gitops-title} 1.14.0
+
+{gitops-title} 1.14.0 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-14.0_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-14-0-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.14.0 security update advisory
+
+Issued: 2024-09-18
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
```

Comment:
I'm sure you know, but need errata info here.

Reply:
I've added Errata information and updated this section.